flake.nix

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
I dont think that nix is very good choice - many developers here use windows and even on linux it is not included in many package managers... I would say that https://github.com/mozilla/web-ext would be more appropriate for packaging this...

Reply:
of course, like I said Nix is there just to ease the development for me, no one else has to to touch it